cluster/sync-common.sh

```diff
@@ -0,0 +1,94 @@
+function eventually {
+    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
+    interval=5
+    cmd=$@
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function consistently {
```

Comment:
Isn't this function always return True no matter what's cmd return value?

Reply:
There is `return 1` if we reach timeout, return 1 is an error at bash.

Reply:
@qinqon  I meant to consistently function, I guess we want to return 1  if $cmd failed.
IIUC, in current code 'consistently' function will return OK (0)

Reply:
@qinqon I meant to consistently function, seems it will always return OK (0) even if $cmd failed